components/opponent/commons/opponent_display.lua

```diff
@@ -209,9 +249,9 @@ OpponentDisplay.propTypes.InlineTeamContainer = {
 	template = 'string',
 }
 
---[[
-Displays a team as an inline element. The team is specified by a template.
-]]
+---Displays a team as an inline element. The team is specified by a template.
+---@param props {flip: boolean?, template: string, style: teamStyle?}
+---@return string?
```

Comment:
How does it return nil?

Reply:
`style?: 'bracket'|'hybrid'|'icon'|'short'|'standard',` in the function that calls this
this function has no return for `'hybrid'|'icon'`, hence the possible nil/empty return